scripts/tests/test_validate_docstrings.py

```diff
@@ -668,6 +668,12 @@ def missing_whitespace_after_comma(self):
         pass
 
 
+class NoDocstrings(object):
```

Comment:
Does this need to be a dedicated class or can we just add this method to the existing class with bad docstrings?

Reply:
It's not a bad docstring It's missing docstring, out of these, each of the classes describe particular bad docstring behaviour: 

* BadGenericDocstrings -- badly phrased docstrings
* BadSummarries -- badly summarized docstrings
* BadParameters -- badly documented parameters
* BadReturns -- bad return docs
* BadSeeAlso -- badly formatted see also section
* BadExamples -- bad examples section

I guess this method might go to BadGenericDocstrings, but none of the candidates for it seem ideal.